config/spotbugs/exclude.xml

```diff
@@ -15,6 +15,10 @@
         <Bug pattern="SIL_SQL_IN_LOOP"/>
         <Class name="io.github.mfvanek.pg.index.health.demo.utils.MigrationsGenerator"/>
     </Match>
+    <Match>
+        <Bug pattern="PRMC_POSSIBLY_REDUNDANT_METHOD_CALLS,SIL_SQL_IN_LOOP"/>
```

Comment:
Why `PRMC_POSSIBLY_REDUNDANT_METHOD_CALLS` was ignored?

Reply:
I need to take foreign keys from DB twice to compare: before applying migrations and after that. SpotBugs produces warning beacause of this.

![image](https://github.com/user-attachments/assets/9fed31a1-16f6-41f7-a0dc-754c1457af63)